packages/manager/apps/web/client/app/hosting/cdn/order/translations/Messages_fr_FR.json

```diff
@@ -1,29 +1,37 @@
 {
   "hosting_cdn_order_title": "Vous allez souscrire l’option Shared CDN",
-  "hosting_cdn_upgrade_title": "Migrer vers l’option Shared CDN",
-
+  "hosting_cdn_order_breadcrumb": "Commande de CDN",
+  "hosting_cdn_upgrade_title": "Changer d'offre Shared CDN",
+  "hosting_cdn_upgrade_breadcrumb": "Changement d'offre de CDN",
+  "hosting_cdn_v1_upgrade_title": "Migrer vers l’option Shared CDN",
+  "hosting_cdn_order_choose_plan": "Choisissez le plan parmi l'un des plans suivants",
```

Comment:
```suggestion
  "hosting_cdn_order_choose_plan": "Choisissez le plan parmi l'un des plans suivants : ",
```

Reply:
Hmm the sentence is used as a `oui-step-form` header, I find it not really nice to have ':' displayed in header form